src/Destructurama.Attributed/Attributed/LogWithNameAttribute.cs

```diff
@@ -38,24 +38,7 @@ public LogWithNameAttribute(string newName)
     /// <inheritdoc/>
     public bool TryCreateLogEventProperty(string name, object? value, ILogEventPropertyValueFactory propertyValueFactory, [NotNullWhen(true)] out LogEventProperty? property)
     {
-        var propValue = propertyValueFactory.CreatePropertyValue(value);
-
-        LogEventPropertyValue? logEventPropVal = propValue switch
```

Comment:
@nblumhardt What was the point to copy objects in that switch block? It looks strange and method should work without it if I have not missed unobvious details. Also I removed `logEventPropVal` null-check since `propertyValueFactory.CreatePropertyValue` always returns some not null `LogEventPropertyValue`.

Reply:
@paymanx Could you please shed light on what is happening in the code here as an author? What is the reason for re-boxing of values?

Reply:
Hi guys. Since I did not receive feedback for a month, I merge this and go on with publishing release. If there are errors, then I will release a fix thereafter.